src/base/TilesRendererBase.js

```diff
@@ -70,6 +70,7 @@ export class TilesRendererBase {
 		this.loadSiblings = true;
 		this.displayActiveTiles = false;
 		this.maxDepth = Infinity;
+		this.showEmptyTiles = false;
```

Comment:
Lets default this to `true` because it's the right way according to the spec. I may want to remove the option at some point but we can leave it for now.

Reply:
Done with b3c03c8eea6c4ef37c25ad534c2cdb8c4415dfd5

Reply:
Thanks! And sorry I thought it yesterday but never typed it.\: `showEmptyTiles` is a little confusing I think. Maybe we can rename the flag `stopAtEmptyTiles`? Or maybe invert the flag and call it `ignoreEmptyTiles`? I'm not sure if you have any preferences

Reply:
Good point! I'll rename it.